src/gordon_gcp/plugins/janitor/gpubsub_publisher.py

```diff
@@ -14,19 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Client module to publish any required DNS changes initiated from
+Plugin module to publish any required DNS changes initiated from
```

Comment:
Hm "client" v "plugin" terminology - I think I'd like someone else ( @matt0bi @descent3me ?) to chime in w/ their opinion: the interfaces are defined as `IPublisherClient`, `IEnricherClient`, etc. They are gordon "clients" to interact with Google products.

I can see the confusion around a gordon "client" (aka implemented interfaces / interface providers) and the API clients we've also defined (`GCEClient`, `GDNSClient`, etc).

Basically, I'd vouch for consistency. So either we keep it as "client" here, or we change the names of the interface definitions.

Reply:
I was actually going to suggest we change those.

Reply:
I'm okay with renaming the interfaces to be plugins, and think that for consistency we shouldn't change the docs until the code changes.

Reply:
I'm for switching the phrasing from client to plugin here - I can see the argument that these are plugin classes (+ they're in the "plugins" directory now).